src/lib/supabaseProfile.ts

```
@@ -0,0 +1,556 @@
+
+  /**
+   * Helper to get user-friendly error message
+   */
+  getErrorMessage(error: SupabaseError | null): string {
+    if (!error) return '';
+
+    const errorType = this.getErrorType(error);
+
+    switch (errorType) {
+      case 'database':
+        const dbError = error as PostgrestError;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)